.drone.yml

```diff
@@ -157,7 +157,7 @@ pipeline:
       - SKELETON_DIR=/mnt/data/apps/testing/data/webUISkeleton
     commands:
       - cd /drone/owncloud-smoketest/tests/acceptance
-      - bash run.sh --remote --tags "@smokeTest&&~@skip" --type webUI
+      - bash run.sh --remote --tags "@smokeTest&&~@skip&&~@skipOnDockerContainerTestin" --type webUI
```

Comment:
s/skipOnDockerContainerTestin/skipOnDockerContainerTesting

Reply:
Fixed - should all be green now 🤞